Changelog.md

```diff
@@ -2,28 +2,28 @@
 
 ## v3.0
 
-- All configuration solder jumpers moved to bottom off the shield for easy access
+- All configuration solder jumpers moved to bottom of the shield for easy access
 - Added alternate relay model SRA-05VDC-SL-C 15A nominal
 
 ## v2.1
 
 - Added possibility to switch digital out 2 -> clock pin (GPIO1/GPIO3)
-- Added header for Auxiliary 5V (required buck converter upgrade to 1000mA)
+- Added header for Auxiliary 5V (requires buck converter upgrade to 1000mA)
 - Wiki updated for shields v2.0 and above
-- Shield mounting holes now is 3mm
+- Shield mounting holes are now 3mm
 
 ## v2.0
 
-- Added copability to connect up to 4 LED strips with level shifting.
-- Added more flexibility with solder on jumpers.
+- Added capability to connect up to 4 LED strips with level shifting.
+- Added more flexibility with solder-on jumpers.
 
 ### Added [terminals add-on](https://github.com/srg74/WLED-wemos-shield/tree/master/resources/Add-ons)
 
 ### Added [linear audio input plug-in, and digital ICS-43434 microphone add-on boards](https://github.com/srg74/WLED-wemos-shield/tree/master/resources/Add-ons) for Shield v1.5 and up
 
 ### v1.51
 
-- Minor update. Fixed library and gerber files errors.
+- Minor update. Fixed library and gerber file errors.
```

Comment:
Gerber file is not single, it's a set of the files

Reply:
As with the header, the text refers to one set of files, hence file, as opposed to files.